gnat2goto/driver/tree_walk.adb

```diff
@@ -679,6 +679,15 @@ package body Tree_Walk is
    is
    begin
       if Ekind (Etype (Name (N))) in Array_Kind then
+         declare
+            Lhs_Type : constant Entity_Id := Etype (Name (N));
+         begin
```

Comment:
Could you leave a comment here explaining what's going on here for future reference?

Reply:
Done.